<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PrabhupadaLoginWindow</class>
 <widget class="QDialog" name="PrabhupadaLoginWindow">
  <property name="modal" stdset="0">
   <bool>false</bool>
  </property>
  <property name="windowTitle" stdset="0">
   <string>Dialog</string>
  </property>
  <property name="windowModality" stdset="0">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="mouseTracking" stdset="0">
   <bool>false</bool>
  </property>
  <property name="geometry" stdset="0">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>263</width>
    <height>255</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="LayoutPrabhupadaLogin">
   <item>
    <layout class="QHBoxLayout" name="LayoutSQL">
     <item>
      <widget class="QRadioButton" name="radioButtonSQLite">
       <property name="text" stdset="0">
        <string>SQLite</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioButtonPostgreSQL">
       <property name="text" stdset="0">
        <string>PostgreSQL</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QFormLayout" name="LayoutConnection">
     <item row="0" column="0">
      <widget class="QLabel" name="laUserName">
       <property name="text" stdset="0">
        <string>User name</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="ComboBoxUserName">
       <property name="editable" stdset="0">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="laPassword">
       <property name="text" stdset="0">
        <string>Password</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="LineEditPassword">
       <property name="echoMode" stdset="0">
        <enum>QLineEdit::Password</enum>
       </property>
       <property name="clearButtonEnabled" stdset="0">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="laDatabase">
       <property name="text" stdset="0">
        <string>Database</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QComboBox" name="ComboBoxDatabaseName">
       <property name="editable" stdset="0">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="laServer">
       <property name="text" stdset="0">
        <string>Server</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QComboBox" name="ComboBoxHostName">
       <property name="editable" stdset="0">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="laPort">
       <property name="text" stdset="0">
        <string>Port</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QComboBox" name="ComboBoxPort">
       <property name="editable" stdset="0">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QCheckBox" name="CheckBoxResetSettings">
       <property name="text" stdset="0">
        <string>Reset settings</string>
       </property>
       <property name="whatsThis" stdset="0">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;This flag resets all settings made by the user!&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons" stdset="0">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::FirstButton|QDialogButtonBox::Ok</set>
     </property>
     <property name="orientation" stdset="0">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>PrabhupadaLoginWindow</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>PrabhupadaLoginWindow</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
